Makefile

```diff
@@ -201,7 +203,7 @@ webapp/.env.local:
 # symlinks where necessary to let it work from the source tree.
 # This stuff is a hack!
 maintainer-install: inplace-install
-inplace-install: build domserver-create-dirs judgehost-create-dirs
+inplace-install-l: build domserver-create-dirs judgehost-create-dirs
```

Comment:
what does this `-l` suffix mean?

Reply:
It stands for (directory) local. The recursive targets all call their `*-l` variant after recursing, see https://www.domjudge.org/docs/manual/8.0/develop.html#makefile-structure

Since I changed `inplace-install` to a recursive target, the original rule here is conflicting and that's why I renamed it to the local one, which is now called automatically.

Reply:
Thanks for explaining and fixing!